src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &--1 :first-child,
+  &--2 :nth-child(-n + 2),
+  &--3 :nth-child(-n + 3),
+  &--4 :nth-child(-n + 4),
+  &--5 :nth-child(-n + 5) {
+    background-image: url(/src/images/star-active.svg);
+  }
+
+  &__star {
+    background-image: url(/src/images/star.svg);
```

Comment:
As with the previous comment, please use a relative path without a leading slash: `background-image: url(../images/star.svg);`.